### ISSUE TITLE
Discussion: do we need the question on the fallback branch

### ISSUE THREAD

**Issue by tsg** [xataio/client-ts#521]
The last question that `xata init` asks is:

```
Choose a default development branch (fallback branch).
    <Create a new branch>
❯   <None>
    main
```

I wonder if we really need question. If the user is just starting with Xata, it's unlikely they will understand the question and take the right decision. I'm thinking to just use `<None>` and the users can add a fallback branch later on when they adopt branching workflows.

**Comment by gimenete** [comment]
My concern if we remove the question is that people will not know about the fallback branch feature and people will use production which is risky. Maybe we can change the question to be more informative such as:

Do you want to use the main branch for development, or do you want to use a different one?
- Use the main branch
- Create and use a different one
- ...

I think it's crucial to consider new users, but it's also important to consider that `xata init` will be used at any point, not just for new users. Maybe I've been using a db in a repo and now I want to use it in another repo. Maybe I'm staring a new database but I've been using xata for a while, etc.

**Comment by tsg** [comment]
In the original form I found the question confusing, especially because  I think `<None>` and `main` would have done the same, so even myself I was getting a bit of choice anxiety :) 

In general, I think we should prioritise the new user experience over the advanced one, and putting myself in the shoes of a new user, I would be confused by the question. I think your reworded question is much better, however, we're not quite there with our branching workflows so it might be too early to recommend people to work in a branch.

In addition, I'm worried about the friction of asking at least 5 questions during `init`, and we've just got started :).
